association-backend/src/association_membership/domain/entities.py

```diff
@@ -50,7 +50,7 @@ class Meta(BaseMeta):
         nullable=False,
     )
 
-    _add_stripe_subscription_payment: List[StripeSubscriptionPayment] = PrivateAttr(
+    _add_payments: List[Union[StripeSubscriptionPayment, PretixPayment]] = PrivateAttr(
```

Comment:
any reason to keep this as private (I mean the underscore)?

Reply:
Mainly because it isn't part of the domain and shouldn't be an part of the domain